Docs: Manage image transformations (#44928)
supabase/supabase: apps/docs/content/guides/storage/serving/image-transformations.mdx

</Admonition>

## Manage image transformations

> You can enable or disable Image Transformations for your project from [Storage Settings](/dashboard/project/_/storage/files/settings) dashboard. Navigate to **Storage** > **Settings** in your project dashboard and toggle the **Enable Image Transformations** option.

Reviewer: ```suggestion
You can enable or disable Image Transformations for your project from the [**Storage** > **Settings**](/dashboard/project/_/storage/files/settings) section of the Dashboard and toggle the **Enable Image Transformations** option.
```

To consolidate this a bit

<Admonition type="note">

> Disabling Image Transformations will prevent any image transformation requests from being processed. This can be useful to prevent unexpected usage or cost if you do not use this feature.

Reviewer: ```suggestion
Disabling Image Transformations prevents any image transformation requests from being processed. This can be useful to prevent unexpected usage or cost if you do not use this feature.
```
